### PR TITLE
6주차 과제 제출 - 양혜원

### PR DESCRIPTION
**Pull request by hhyewon**
### PR 올리기 전에 확인사항

1. head barnch와 sync를 맞춰주세요.

```bash
$ git pull https://github.com/hanghae-plus/front_2nd.git main
```

2. base branch를 `hanghae-plus:main`이 아니라 `hanghae-plus:<본안아이디>` 로 수정해주세요 확인해주세요.

### CODE REVIEW

**Comment by spartacontents** [comment]
## 🎉랜덤 리뷰 
 랜덤 리뷰로 선정되신 @bbbjihan님!🎉 리뷰를 부탁드립니다.

**Comment by spartacontents** [comment]
## ❌ Build Failed 
 TSC or Build process failed. Please check the errors and fix them.

**Comment by spartacontents** [comment]
## 테스트 성공✅ 
  
 ### 커버리지 결과 


  • 라인 커버리지: 82.74%
  • 브랜치 커버리지: 92.39%
  • 함수 커버리지: 86.95%
  • 구문 커버리지: 82.74%

**Comment by spartacontents** [comment]
## Lighthouse 성능 측정 결과 
 ⚡️ Lighthouse report 0
| Category | Score |
| --- | --- |
| 🟠 Performance | 77 |
| 🟠 Accessibility | 84 |
| 🟢 Best practices | 96 |
| 🟠 SEO | 0.82 |


⚡️ Lighthouse report 1
| Category | Score |
| --- | --- |
| 🟢 Performance | 92 |
| 🟠 Accessibility | 84 |
| 🟢 Best practices | 100 |
| 🟠 SEO | 0.82 |


⚡️ Lighthouse report 2
| Category | Score |
| --- | --- |
| 🟢 Performance | 92 |
| 🟠 Accessibility | 84 |
| 🟢 Best practices | 100 |
| 🟠 SEO | 0.82 |

**Comment by spartacontents** [comment]
## 🎉랜덤 리뷰 
 랜덤 리뷰로 선정되신 @bbbjihan님!🎉 리뷰를 부탁드립니다.

**Comment by spartacontents** [comment]
## ESLint Results 
 > assignment-6@0.0.0 lint /home/runner/work/front_2nd/front_2nd/packages/assignment-6
> eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0 --format stylish 


/home/runner/work/front_2nd/front_2nd/packages/assignment-6/src/forLintError.ts
  2:3  warning  Unexpected console statement  no-console

/home/runner/work/front_2nd/front_2nd/packages/assignment-6/src/main.tsx
  4:10  error  'forLintError' is defined but never used  @typescript-eslint/no-unused-vars

✖ 2 problems (1 error, 1 warning)

/home/runner/work/front_2nd/front_2nd/packages/assignment-6:
 ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  assignment-6@0.0.0 lint: `eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0 --format stylish `
Exit status 1

**Comment by spartacontents** [comment]
## ❌ Build Failed 
 TSC or Build process failed. Please check the errors and fix them.

**Comment by spartacontents** [comment]
## 테스트 성공✅ 
  
 ### 커버리지 결과 


  • 라인 커버리지: 82.74%
  • 브랜치 커버리지: 92.39%
  • 함수 커버리지: 86.95%
  • 구문 커버리지: 82.74%

**Comment by spartacontents** [comment]
## Lighthouse 성능 측정 결과 
 ⚡️ Lighthouse report 0
| Category | Score |
| --- | --- |
| 🟢 Performance | 92 |
| 🟠 Accessibility | 84 |
| 🟢 Best practices | 96 |
| 🟠 SEO | 82 |


⚡️ Lighthouse report 1
| Category | Score |
| --- | --- |
| 🟢 Performance | 93 |
| 🟠 Accessibility | 84 |
| 🟢 Best practices | 100 |
| 🟠 SEO | 82 |


⚡️ Lighthouse report 2
| Category | Score |
| --- | --- |
| 🟢 Performance | 92 |
| 🟠 Accessibility | 84 |
| 🟢 Best practices | 100 |
| 🟠 SEO | 82 |

File: .github/workflows/assignment-6-ci.yml [hunk 1]
```diff
@@ -2,38 +2,243 @@ name: CI Assignment 6
...
+      - name: Run tests
+        run: pnpm -F assignment-6 test:coverage
+
+      - name: test pwd
+        run: pwd
+
+      - name: Comment test results
+        if: success()
+        uses: actions/github-script@v6
+        with:
```
Comment (JunilHwang) [inline .github/workflows/assignment-6-ci.yml:124]: 테스트가 실패하는 경우에도 알려주면 좋을 것 같아요!

File: .github/workflows/assignment-6-ci.yml [hunk 2]
```diff
@@ -2,38 +2,243 @@ name: CI Assignment 6
...
     steps:
-      - name: job5 구현
-        run: echo "job5의 내용을 구현해주세요"
+      - name: assign random reviewer
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ env.GITHUB_TOKEN }}
+          script: |
+            const teams = ["bbbjihan", "soojjung"];
+            const randomReviewer = teams[Math.floor(Math.random() * teams.length)];
```
Comment (JunilHwang) [inline .github/workflows/assignment-6-ci.yml:230]:
```tsx
const reviewer = ["bbbjihan", "soojjung"].sort(() => Math.random() - 0.5).pop()
```
이렇게 표현할 수도 있답니다!

**Pull request merged**